docs/issues.md

```diff
@@ -0,0 +1,26 @@
+# Filing An Issue
+The Openshift Aggregated Logging integration is a multicomponent technology stack whose individual parts must all work together to gather and archive logs.  It is important to include as much information as possible when issues arise in order to properly debug and diagnose.  Run **ADD SCRIPT OR UTILITY HERE** which will gather the desired information.  Alternatively, you can gather them manually.  Please include as much of the following to help us expedite a resolution:
+
+### OpenShift Cluster Details
+1. Cluster version (e.g. OCP v3.5)
+2. Cluster infrastructure (e.g. AWS, GCP)
+3. Number of infrastructure nodes and size (e.g. CPU, RAM)
+4. Number of compute nodes and size (e.g. CPU, RAM)
+5. Logging Image versions
+
+**ADD NOTE HERE ON HOW TO RETRIEVE IMAGE VERSION FROM OCP AND ORIGIN**
+
+### Installer Details
+1. How was logging installed (e.g. Openshift Ansible/Installer, deployer)
+2. Openshift installer version
+3. Ansible inventory file
+4. Ansible installer logs
+
+### Logging Component Details
+**MAYBE WE CAN GRAB SOME OF THESE BY LABEL**
+1. Elasticsearch logs: `SOME NON-OC LOGS COMMAND`
+2. Other component logs: `oc logs $PODNAME`
+3. Fluentd spec: `oc get ds logging-fluentd -o yaml`
+4. Other component specs: `oc get dc $DCNAME -o yaml`
```

Comment:
Is that different then the DC?  Should the resulting POD be what is defined in the DC/DS?  Do we want both?